i18n/en.json

```diff
@@ -344,6 +346,7 @@
   "admin.billing.subscription.privateCloudCard.cloudStarter.description": "Optimize your processes with Guest Accounts, Office365 suite integrations, GitLab SSO and advanced permissions.",
   "admin.billing.subscription.privateCloudCard.cloudStarter.title": "Upgrade to Cloud Professional",
   "admin.billing.subscription.privateCloudCard.contactSales": "Contact Sales",
+  "admin.billing.subscription.privateCloudCard.contactSalesy": "Contact sales",
```

Comment:
@annelieseklein could the action button have the `Contact Sales` title case? It seems this is how it is presented in other buttons. Not sure we would need another text just for this